ddtrace/span.py

```diff
@@ -123,7 +123,7 @@ def __init__(
         # required span info
         self.name = name
         self.service = service
-        self.resource = resource or name
+        self._resource = [resource or name]
```

Comment:
Should the empty string be treated as `None`?

Reply:
I do not think I can answer this question for tracing.
I think as far as profiling goes we convert nones to strings for pprof.